js_modules/dagit/packages/core/src/launchpad/ConfigEditorConfigPicker.tsx

```diff
@@ -405,6 +462,13 @@ const CONFIG_PARTITIONS_QUERY = gql`
         }
       }
     }
+    assetNodes(assetKeys: $assetKeys) {
```

Comment:
@bengotow On second thought, should I be using `usePartitionHealthData` here instead?